README.md

```diff
@@ -89,15 +69,31 @@ tags = ['version:1', 'application:web']
 api.Event.create(title=title, text=text, tags=tags)
 ```
 
-Thread Safety
--------------
-`DogStatsD` and `ThreadStats` are thread-safe.
+## DogStatsD
+
+In order to use DogStatsD metrics, the Agent must be [running and available](https://docs.datadoghq.com/developers/dogstatsd/?tab=python).
+
+### Instantiate the DogStatsD client
+
+Once the Datadog Python Library is installed, instantiate the StatsD client in your code:
+
+```python
+from datadog import statsd
+
+options = {
+    'statsd_host':'127.0.0.1',
+    'statsd_port':8125
+}
+
+initialize(**options)
```

Comment:
🤔 we could indeed, but it's in case people won't click on the link. At least they have the minimum configuration required for it to work.

It's also to avoid this page to be just a summary of links which could create a "Wall of text" effect i think.

Reply:
ok :+1: